pmdarima/arima/tests/test_arima.py

```diff
@@ -24,6 +24,25 @@
 import time
 import os
 
+# test images directories
+travis = os.environ.get("TESTING_ON_TRAVIS", "false").lower() == "true"
+
+if not travis:
+
+    # Only do this setup if we are not running on Travis CI.
+    from matplotlib.pyplot import savefig
+    from matplotlib.testing.exceptions import ImageComparisonFailure
+    import matplotlib._png as _png
+
+    test_images_input_path = \
+        os.path.join(os.path.dirname(__file__), 'input_images')
+    test_images_output_path = \
+        test_images_input_path.replace('input_images', 'output_images')
```

Comment:
Can you wrap these in `try`/`finally` so that we always nuke the .png files at the end?

Reply:
Done!